invenio_cli/helpers/scripts.py

```diff
@@ -0,0 +1,298 @@
+    if base:
+        click.secho('Building base docker image...', fg='green')
+        # docker build -f Dockerfile.base -t my-site-base:latest .
+        docker_helper.built_image(
+            dockerfile='Dockerfile.base',
+            tag='{project_name}-base:latest'.format(
+                project_name=app_name)
+        )
+    if app:
+        click.secho('Building applications docker image...', fg='green')
+        # docker build -t my-site:latest .
+        docker_helper.built_image(
+            dockerfile='Dockerfile',
+            tag='{project_name}:latest'.format(
+                project_name=app_name)
+        )
+
+
+def bootstrap(dev=True, pre=True,
+              base=True, app=True, docker_helper=None, app_name='invenio-rdm',
+              verbose=False, loglevel=logging.WARN, logfile='invenio-cli.log'):
+    """Bootstrap server."""
+    click.secho('Bootstrapping server...', fg='green')
+    if dev:
+        _bootstrap_dev(pre, loglevel, logfile, verbose)
+    else:
+        _boostrap_prod(base, app, docker_helper, app_name)
+
+
+@with_appcontext
+def build_assets(verbose):
+    """Build assets."""
+    cli = create_cli()
+    runner = current_app.test_cli_runner()
+    # Collect
+    run_command(cli, runner, "collect -v", message="Collecting assets...",
+                verbose=verbose)
+
+    # Build using webpack
+    run_command(cli, runner, "webpack buildall",
+                message="Building assets...", verbose=verbose)
+
+
+def _force_symlink(file1, file2):
+    try:
+        os.symlink(file1, file2)
+    except OSError as e:
+        if e.errno == errno.EEXIST:
+            os.remove(file2)
+            os.symlink(file1, file2)
+
+
+def update_config(dev, loglevel, logfile):
+    """Update invenio.cfg configuration file."""
+    if dev:
+        # Create symbolic link
+        dst_path = _get_instance_path(loglevel, logfile)
+        src_file = os.path.abspath('invenio.cfg')
+        _force_symlink(src_file, Path(dst_path) / 'invenio.cfg')
+
+    else:
+        # Copy to container
+        pass
+
+
+def update_statics(dev, loglevel, logfile):
+    """Update static files."""
+    if dev:
+        # Copy logo file
+        dst_path = _get_instance_path(loglevel, logfile)
+        dst_path = Path(dst_path) / 'static/images'
+        src_file = os.path.abspath('static/images/logo.svg')
+        click.secho("Creating statics folder...", fg="green")
+        try:
+            os.makedirs(dst_path)  # Create directories if doesnt exist
+        except FileExistsError:
+            click.secho("Statics directory already exsits...", fg="yellow")
+
+        shutil.copyfile(src_file, dst_path / 'logo.svg')
+
+    else:
+        pass
+
+
+def _get_instance_path(loglevel, logfile):
+    # Open logging pipe
+    logpipe = LogPipe(loglevel, logfile)
+
+    calculate_path = subprocess.Popen(
+        ['pipenv', 'run', 'invenio', 'shell', '--no-term-title',
+            '-c', '"print(app.instance_path)"'],
```

Comment:
```suggestion
            '-c', '"print(app.instance_path, end='')"'],
```

This should remove the need to remove the `\n` . Why do we need to `decode('utf-8')?